.github/workflows/stale-pr.yml

```diff
@@ -4,8 +4,14 @@ on:
   # * is a special character in YAML so you have to quote this string
   - cron: "0 0 * * *"
 
+permissions:
+  contents: read
+
 jobs:
   stale:
+    permissions:
+      issues: write  # for actions/stale to close stale issues
```

Comment:
We are not closing the prs, the not only adds a label stale

Reply:
Hi @phofl, is your ask to remove or change the comment? Just wanted to confirm before making the change. 

I can remove the comment or change it to `to label stale PRs`. 

Also, looks like this workflow does not update stale issues. If that is the case, I can remove the `issues: write` permission.

Reply:
This action only adds a `stale` label and writes one comment to a PR, so whatever permission are needed to perform those actions can be set.

Reply:
Updated `stale-pr.yml`. Removed `issues: write` since issues are not being updated based on the config. Also removed the comment for `pull-requests: write`.